src/CoreWCF.Primitives/src/CoreWCF/Security/SecuritySessionServerSettings.cs

```diff
@@ -1985,7 +2552,7 @@ protected override Task OnReplyAsync(Message message, CancellationToken token)
                 if (message != null)
                 {
                     _channel.SecureApplicationMessage(ref message, _correlationState, token);
-                    //this.securityProtocol.SecureOutgoingMessageAsync(message);
+                    //securityProtocol.SecureOutgoingMessageAsync(message);
                     return _requestContext.ReplyAsync(message);
```

Comment:
Can you remove the old code as it looks like _channel.SecureApplicationMessage calls SecureOutgoingMessageAsync